src/seedu/addressbook/data/person/Person.java

```diff
@@ -24,6 +26,7 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.phone = phone;
         this.email = email;
         this.address = address;
+        this.sequenceNumber = ++nextSequenceNumber;
```

Comment:
You have assigned nextSequenceNumber to be 1 and you are incrementing nextSequenceNumber before you are assigning it to sequenceNumber. So basically aren't you missing number 1 in your sequence?
Hint: Check the difference between ++nextSequenceNumber and nextSequenceNumber++